.gitignore

```diff
@@ -49,3 +49,6 @@ tools/generated/*
 
 # generated ssl certs
 compose/ingress/certs/*
+
+# static files
+test_app/static_collected/
```

Comment:
I put this in /static/ first, but someone had already put 1 file in that folder. Using a different dir requires more settings to manage that but is then non-conflicting with this existing file. I tested it and things are working.